sql/planbuilder/ddl.go

```diff
@@ -1032,6 +1032,7 @@ func (b *Builder) tableSpecToSchema(inScope, outScope *scope, db sql.Database, t
 
 	defaults := make([]ast.Expr, len(tableSpec.Columns))
 	generated := make([]ast.Expr, len(tableSpec.Columns))
+	updates := make([]ast.Expr, len(tableSpec.Columns))
```

Comment:
did we only need this for update default? should we have a separate test for this if there is a non-default query that errors without it?

Reply:
Oops this was leftover from an attempt to `OnUpdate` functionality; will remove